README.md

```diff
@@ -47,12 +47,12 @@ let mut u = Username::new(" valid name \n\n").unwrap();
 assert_eq!(u.get(), "valid name"); // now we're talking!
 
 // This also works for mutations:
-assert!(matches!(u.try_mutate(|u| *u = "   ".to_owned()), Err(prae::ValidationError)));
+assert!(matches!(u.try_mutate(|u| *u = "   ".to_owned()), Err(prae::ConstructionError<String>)));
```

Comment:
Shouldn't it be `MutationError<String>`?

Reply:
Yea I didn't add the mutation error yet

Reply:
I want to argue that the generic bound `E` should be removed. See the comment below.